class/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```

Comment:
Nice job so far😉. Just as I have explained in the Student class,  it is also a requirement here that the constructor extends the parent's constructor by adding @specialization and a parameter for it😉.

Kindly do well to take a look at how I implemented it in the suggestion above and try to make the necessary change👍🏼